### ISSUE TITLE
Typo: "Solutions for erratic curves"      +learning_rate=1e-4  must be +learning_rate=1e-46

### ISSUE THREAD

**Issue by AhmedMagdi2003** [huggingface/course#1240]
in the code provided to sloution to erratic by decreasing the learning rate, but 
the code increase it
'''
from transformers import TrainingArguments

training_args = TrainingArguments(
    output_dir="./results",
    -learning_rate=1e-5,
    +learning_rate=1e-4,
    -per_device_train_batch_size=16,
    +per_device_train_batch_size=32,
)
